### PR TITLE
Add REST function call timeout

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Add REST function call timeout
- Not require service config in _createApp()
- Add tool scripts for chatbot
- Remove transaction-executor code

Related issue: https://github.com/ainblockchain/ain-blockchain/issues/364

### CODE REVIEW

File: tools/chatbot/sendMessageTx.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  console.log(`txInfo: ${JSON.stringify(txInfo, null, 2)}`);
+  if (!txInfo.success) {
+    console.log(`Message transaction failed.`);
+    process.exit(0);
+  }
+  await confirmTransaction(config.endpointUrl, timestamp, txInfo.txHash);
+}
+
+async function processArguments() {
+  if (process.argv.length !== 3 && process.argv.length !== 4) {
```
Comment (cshcomcom) [inline tools/chatbot/sendMessageTx.js:34]: `if (process.argv.length !== 3 || process.argv.length !== 4)`?

Comment (cshcomcom) [reply tools/chatbot/sendMessageTx.js:34]: My mistake! Please ignore this comment

**Comment by cshcomcom** [comment]
I think 'DApp' test was something wrong... (`Iteration count exceeded its limit before the given tx 0x420672bc23cb378657be32408b1c9778d36ee1311875d2b676e092d5879f9778 is finalized!` messages)

image

**Review by cshcomcom — APPROVED** [review]
LGTM! Thanks

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

**Pull request merged**